### PR TITLE
Primeros cambios a sufia.es.yml para su traducción

### PR DESCRIPTION
**Pull request by ejrivero**
Fixes #issuenumber ; refs #issuenumber

Present tense short summary (50 characters or less)

More detailed description, if necessary. Try to be as descriptive as you can: even if you think that the PR content is obvious, it may not be obvious to others. Include tracebacks if helpful, and be sure to call out any bits of the PR that may be work-in-progress.

Description can have multiple paragraphs and you can use code examples inside:

``` ruby
class PostsController
  def index
    respond_with Post.limit(10)
  end
end
```

Changes proposed in this pull request:
- 
- 
- 

@projecthydra/sufia-code-reviewers

Primeros cambios a sufia.es.yml para su traducción, aún faltan bastantes términos

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Thank you, @ejrivero! I am reviewing with the Hydra Steering group whether YAML contributions require contributor license agreements. I'll reply again as soon as I know more.

:clap:

**Comment by wickr** [comment]
@ejrivero @mjgiarlo looks like there's a duplicate set of keys, starting at line #394 they start repeating, with a new :es and :sufia and more.

**Comment by mjgiarlo** [comment]
Good call, @wickr. @ejrivero: would you mind taking a look at that and pushing a change to your branch?

**Comment by mjgiarlo** [comment]
@ejrivero Are there any updates to share re: licensing agreements? We're eager to get your PR merged. :smile:

File: config/locales/sufia.es.yml [hunk 1]
```diff
@@ -1,4 +1,443 @@
...
+        tab_label:          'Explorar Colecciones'
+        title:              'Explorar Colecciones'
+    toolbar:
+      admin:
+        menu: "Administración"
+      dashboard:
+        menu: "Panel de Control"
+        my: "Mi Panel de Control"
+        highlights: "Aspectos destacados"
+        transfers: "Transferirencias"
```
Comment (luisgreg99) [inline config/locales/sufia.es.yml]: Did you mean "Transferencias"?

File: config/locales/sufia.es.yml [hunk 2]
```diff
@@ -1,4 +1,443 @@
...
+        shares: "Compartido"
+      works:
+        menu: "Trabajos"
+        new: "Nuevo"
+        my: "Mi Trabajos"
```
Comment (luisgreg99) [inline config/locales/sufia.es.yml]: I think this would be "Mis Trabajos"

File: config/locales/sufia.es.yml [hunk 3]
```diff
@@ -1,4 +1,443 @@
...
+      no_activity:              "El Usuario no tiene actividad reciente"
+      user_notifications:       "Notificaciones de Usuario"
+      no_notifications:         "El Usuario no tiene notificaciones"
+      additional_notifications: "ver todas las notificaciones"
+      manage_proxies:           "Administrar Proxies"
+      proxy_activity:           "Actividad de Proxy"
+      authorize_proxies:        "Autorizar Proxies"
+      current_proxies:          "Proxies actuales"
+      proxy_user:               "Usuario de Proxy"
+      transfer_of_ownership:    "Transferirencia de propiedad"
```
Comment (luisgreg99) [inline config/locales/sufia.es.yml]: I think "Transferencia de propiedad" looks better :)

File: config/locales/sufia.es.yml [hunk 4]
```diff
@@ -1,4 +1,443 @@
...
+      progress:
+        header: Guardar Trabajo
+      show:
+        no_preview: No hay vista previa disponible
+    batch_uploads:
+      new:
+        header: Crear nuevos Trabajos en lote
+        in_collections: Estos Trabajos en Colecciones
+        in_this_work: Otros Trabajos en este Trabajo
+        in_other_works: Este Trabajo en otrosTrabajos
```
Comment (luisgreg99) [inline config/locales/sufia.es.yml]: Missing space character: "Este Trabajo en otros Trabajos"

**Comment by mjgiarlo** [comment]
I chatted with Alberto today and he thinks they'll be able to submit signed CLAs by the end of this week.

**Comment by mjgiarlo** [comment]
@ejrivero What do you think of @luisgreg99's suggestions above?

File: config/locales/sufia.es.yml (hunk 1 above)
Comment (ejrivero) [inline config/locales/sufia.es.yml]: Yeah, thanks.

**Comment by mjgiarlo** [comment]
This PR is now unblocked; @ejrivero is now a licensed Hydra contributor!

**Comment by mjgiarlo** [comment]
I'm porting this to Hyrax.

**Comment by mjgiarlo** [comment]
Replaced by https://github.com/projecthydra-labs/hyrax/pull/33. Thanks, @ejrivero!

**Pull request closed without merging**